_includes/menu.html

```diff
@@ -1,10 +1,9 @@
 <ul class="header-ul">
 <li><a href="/index.html">Home</a></li>
-<li><a href="assets/data/MDD.zip">Download the Database</a></li>
-<li><a href="taxa.html">Explore Taxonomy</a></li>
-<li><a href="tree.html">Treeview</a></li>
-<li><a href="explore.html">Search Species</a></li>
-<li><a href="linksToData.html">Links to Mammal Data</a></li>
-<li style="float:right"><a href="about.html">About</a></li>
+<li><a href="/assets/data/MDD.zip">Download the Database</a></li>
```

Comment:
@JelleZijlstra why replace the relative with the absolute links? Was the functionality broken somehow?

Reply:
If you go to https://www.mammaldiversity.org/taxon/1000001 and click one of the links in the header, it goes to e.g. https://www.mammaldiversity.org/taxon/taxa.html.

Reply:
ok, thanks for pointing this out! Suggest to use jekyll variable to point to root of the site instead. See e.g.,  https://talk.jekyllrb.com/t/relative-url-and-baseurl/2051 . Pointing to the root of the domain may cause issues when deploying in subdomains / subpaths.

Reply:
Good point, done.